gui/wxpython/main_window/frame.py

```diff
@@ -179,30 +179,30 @@ def show_menu_errors(messages):
 
         # use default window layout ?
         if UserSettings.Get(group="general", key="defWindowPos", subkey="enabled"):
-            dim = UserSettings.Get(group="general", key="defWindowPos", subkey="dim")
-            try:
-                x, y = map(int, dim.split(",")[0:2])
-                w, h = map(int, dim.split(",")[2:4])
-                client_disp = wx.ClientDisplayRect()
-                if x == 1:
-                    # Get client display x offset (OS panel)
-                    x = client_disp[0]
-                if y == 1:
-                    # Get client display y offset (OS panel)
-                    y = client_disp[1]
-                self.SetPosition((x, y))
-                self.SetSize((w, h))
-            except Exception:
-                pass
-            self.Layout()
-            if w <= globalvar.GM_WINDOW_SIZE[0] or h <= globalvar.GM_WINDOW_SIZE[1]:
-                self.Fit()
-        else:
-            self.Layout()
-            self.Fit()
-            # does center (of screen) make sense for lmgr?
-            self.Centre()
-
+            single_window_dim = {
+                "group": "general",
+                "key": "defWindowPos",
+                "subkey": "dimSingleWindow",
+            }
+            dim = UserSettings.Get(**single_window_dim)
+            default_dim = UserSettings.Get(**single_window_dim, settings_type="default")
+            if dim != default_dim:
+                try:
+                    x, y = map(int, dim.split(",")[0:2])
+                    w, h = map(int, dim.split(",")[2:4])
+                    client_disp = wx.ClientDisplayRect()
+                    if x == 1:
+                        # Get client display x offset (OS panel)
+                        x = client_disp[0]
+                    if y == 1:
+                        # Get client display y offset (OS panel)
+                        y = client_disp[1]
+                    self.SetPosition((x, y))
+                    self.SetSize((w, h))
+                except Exception:
+                    pass
+            else:
+                self.Maximize(True)
```

Comment:
`self.Maximize()`is never reached in a "first run" simulation (with no existing`~/.grass8` dir), placing the window partially under the OS menu on Mac:

<img width="282" alt="Screenshot_default" src="https://github.com/OSGeo/grass/assets/14186207/aa3b857c-3c6e-402d-8df0-67963b39e4dd">


Outdenting the `else: self.Maximize()´ seems to work for me :

```py
        if UserSettings.Get(group="general", key="defWindowPos", subkey="enabled"):
            single_window_dim = {
                "group": "general",
                "key": "defWindowPos",
                "subkey": "dimSingleWindow",
            }
            dim = UserSettings.Get(**single_window_dim)
            default_dim = UserSettings.Get(**single_window_dim, settings_type="default")
            if dim != default_dim:
                try:
                    x, y = map(int, dim.split(",")[0:2])
                    w, h = map(int, dim.split(",")[2:4])
                    client_disp = wx.ClientDisplayRect()
                    if x == 1:
                        # Get client display x offset (OS panel)
                        x = client_disp[0]
                    if y == 1:
                        # Get client display y offset (OS panel)
                        y = client_disp[1]
                    self.SetPosition((x, y))
                    self.SetSize((w, h))
                except Exception:
                    pass
        else:
            self.Maximize(True)
```